### PR TITLE
add equivalent of `depends_on_past` and `task_concurrency` to the dagster-airflow page

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

File: docs/content/integrations/airflow.mdx [hunk 1]
```diff
@@ -20,22 +20,25 @@ This integration is designed to help support users who have existing Airflow usa
...
+| `depends_on_past`     | An asset can [depend on earlier partitions of itself](https://github.com/dagster-io/dagster/discussions/11829). When this is the case, [backfills](/concepts/artitions-schedules-sensors/backfills) and [auto-materialize](/concepts/assets/asset-auto-execution) will only materialize later partitions after earlier partitions have completed. |                                                                                                                                                                                                       |
+
+| `task_concurrency` | [Op/asset-level concurrency limits](https://docs.dagster.io/guides/limiting-concurrency-in-data-pipelines#limiting-opasset-concurrency-across-runs) ]
```
Comment (erinkcochran87) [inline docs/content/integrations/airflow.mdx:44]:
```suggestion
| `task_concurrency` | [Op/asset-level concurrency limits](/guides/limiting-concurrency-in-data-pipelines#limiting-opasset-concurrency-across-runs) ]
```

File: docs/content/integrations/airflow.mdx [hunk 2]
```diff
@@ -20,22 +20,25 @@ This integration is designed to help support users who have existing Airflow usa
...
+| Datasets              | [Software-defined assets (SDAs)](/concepts/assets/software-defined-assets)                                                                                                                                                                                                                                                                        | SDAs are more powerful and mature than datasets and include support for things like partitioning.                                                                                                     |
+| Connections/Variables | [Run config](/concepts/configuration/config-schema), [Configured API](/concepts/configuration/configured) and [environment variables](https://docs.dagster.io/dagster-cloud/developing-testing/environment-variables-and-secrets#dagster-cloud-environment-variables-and-secrets) (Dagster Cloud only)                                            |                                                                                                                                                                                                       |
```
Comment (erinkcochran87) [inline docs/content/integrations/airflow.mdx:41]:
```suggestion
| Connections/Variables | [Run config](/concepts/configuration/config-schema), [Configured API](/concepts/configuration/configured) and [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets#dagster-cloud-environment-variables-and-secrets) (Dagster Cloud only)                                            |                                                                                                                                                                                                       |
```

**Review by erinkcochran87 — COMMENTED** [review]
This looks ok to me! I left two comments about some internal links.

File: docs/content/integrations/airflow.mdx (hunk 1 above)
Comment (sryza) [reply docs/content/integrations/airflow.mdx:44]: Oops

File: docs/content/integrations/airflow.mdx (hunk 2 above)
Comment (sryza) [reply docs/content/integrations/airflow.mdx]: I tried making this change, but the build is claiming that the link is dead after I do that. So maybe there's a reason this was put there before? This was already here before the change, proposed in this PR, so I think best to leave it be here.

**Pull request merged**